Ecommerce/ProductService/ProductService.csproj

```
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <Nullable>enable</Nullable>
```

Если ты нигде не используешь суффикс `?`, то можешь заменить на
```suggestion
    <Nullable>disable</Nullable>
```